.github/workflows/test-unit-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
 
       - run: |
-          cargo test --workspace \
+          cargo test --workspace --all-features \
```

Comment:
Needed because `driver-adapters` crate has no default features and therefore does not include any connectors by default.